spec/string_validator_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'string_validator'
+
+RSpec.describe StringValidator do
```

Comment:
Do remember to add in your sad path tests and edge case tests. They may not be one hundred percent useful, but it shows people that you're thinking ahead and thinking about possibilities. It makes people assume that you're a lot deeper than you may be.